src/routes/user.js

```diff
@@ -34,32 +28,56 @@ router.get('/users/:id', async(req, res) => {
     }
 })
 
-router.patch('/users/update/:id', async(req, res) => {
-     const updates = Object.keys(req.body)
-     const allwoedUpdates = ['name', 'password', 'age']
-     const isValidOperation = updates.every((update) => {
-         return allwoedUpdates.includes(update)
-     })
+// router.patch('/users/update/:id', async(req, res) => {
```

Comment:
Whenever I try to validate the ID, the terminal gave me something like req.params is undefiend